datasafe-business/src/test/java/de/adorsys/datasafe/business/impl/e2e/StorageBasedVersioningTest.java

```diff
@@ -28,6 +29,7 @@
     private static final String FILE = "file.txt";
 
     @Test
+    @DisabledIfSystemProperty(named = SKIP_CEPH, matches = "true")
```

Comment:
Why it is on each method instead of being on class?

Reply:
done